odk2odm/odk_api.py

```diff
@@ -91,17 +111,89 @@ def attachment(base_url, aut, projectId, formId, instanceId, filename):
         f'{instanceId}/attachments/{filename}'
     return requests.get(url, auth = aut)
 
+# POST 
 
 def create_project(base_url, aut, project_name):
     """Create a new project on an ODK Central server"""
     url = f'{base_url}/v1/projects'
     return requests.post(url, auth = aut, json = {'name': project_name})
 
+def create_app_user(base_url, aut, projectId, app_user_name = 'Surveyor'):
```

Comment:
Why is app_user_name defaulted to 'Surveyor'? Can we not simply pick a good other name?